src/Entities/Player.cpp

```diff
@@ -1203,6 +1203,7 @@ void cPlayer::KilledBy(TakeDamageInfo & a_TDI)
 					case dtAdmin:           return "was administrator'd";
 					case dtExplosion:       return "blew up";
 					case dtAttack:          return "was attacked by thin air";
+					case dtEnvironment:     return "played to much dress up";  // This is not vanilla - added a own pun
```

Comment:
you have to explain the pun, I don't get it...

Reply:
You only get this type of damage if you are an enderman in the rain or a snowman in the desert

Reply:
I prefer "succumbed to the conditions", but your choice would be fine after fixing the typo "to" -> "too".